extensions/markdown-language-features/src/languageFeatures/documentLinkProvider.ts

```diff
@@ -173,7 +173,7 @@ const referenceLinkPattern = /(^|[^\]\\])(?:(?:(\[((?:\\\]|[^\]])+)\]\[\s*?)([^\
 /**
  * Matches `<http://example.com>`
  */
-const autoLinkPattern = /\<(\w+:[^\>\s]+)\>/g;
+const autoLinkPattern = /\<(\w+:\/{1,}[^\>\s]+)\>/g;
```

Comment:
Given that `+` is already used in this pattern...
```suggestion
const autoLinkPattern = /\<(\w+:\/+[^\>\s]+)\>/g;
```